### ISSUE TITLE
Angled walls 

### ISSUE THREAD

**Issue by TfedUD** [ladybug-tools/dragonfly-doe2#42]
Need to add capability for tilted walls, should be able to use the same MO as the shading geometry

**Comment by mostaphaRoudsari** [comment]
I want to rephrase this as the ability to handle HBJSON rooms. In other words, we can draw all the faces with this new methodology which can also handle angled walls.

What you wrote there gives me the feeling that you want to check for every wall if they are angled and then use a different method.
